dev/com.ibm.ws.webservices.javaee.common/bnd.bnd

```diff
@@ -24,6 +24,7 @@ Private-Package: \
 
 Import-Package: \
  javax.xml.ws.*;version="[2.2,3)", \
+ com.ibm.ws.jaxws.* ;version="[1.0.0,3)", \
```

Comment:
Why 1.0.0 instead of 2.0 like you had in the other bnd files?

Reply:
Fixed